safedata_validator/summary.py

```diff
@@ -28,6 +29,55 @@
 RE_NAME = re.compile(r"[^,]+,[ ]?[^,]+")
 RE_CONTAINS_WSPACE = re.compile(r"\s")
 
+# Class attribute to define the metadata that may be present.
+# These are defined in blocks of related rows described in 4-tuples:
+#  0: a list of tuples giving the rows in the block:
+#     * row header key
+#     * mandatory within the block
+#     * 'internal' name - also maps to Zenodo fields
+#     * accepted types
+#     * list of aliases for the fieldname (this generally will be blank)
```

Comment:
As far as I can tell this isn't used anymore. Am I right in my impression that all the (pretty convoluted) generic alias handling is gone, in favour of explicitly handling the "project id" vs "safe project id" case

Reply:
That's right - forgot to mention that and it was quite a specific change. All of the 'alias' handling has been deleted in favour of a specific project block that handles the two cases within a dedicated `_load_project_ids` method. I thought it was unlikely we'd deliberately allow more aliases and I had to create a new block to allow multiple project ids.

Reply:
Actually, that _whole_ block comment is a mistake from creating those two new dataclasses!